containers/proxy-httpd-image/uyuni-configure.py

```diff
@@ -93,6 +93,13 @@ def getIPs(fqdn: str) -> Tuple[str, str]:
     with open("/etc/rhn/rhn.conf", "w") as file:
         file.write(f'''# Automatically generated Uyuni Proxy Server configuration file.
         # -------------------------------------------------------------------------
+        
+        # Debug log level
+        debug = {config.get("debug_log_level", 4)}
```

Comment:
So by default we want debug log enabled? Or do you later plan to lower it?

Reply:
default should be set to 1 as in the default config